Google.Api.Gax.Grpc.Tests/Rest/PartialDecodingStreamReaderTest.cs

```diff
@@ -0,0 +1,165 @@
+﻿/*
+ * Copyright 2022 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Google.Api.Gax.Grpc.Rest.Tests
+{
+    public class PartialDecodingStreamReaderTest
+    {
+        private static readonly string DataStr;
```

Comment:
Please rename to something more descriptive - and personally I'd suggest using a verbatim string literal to show exactly what you expect it to look like.